applications/crossbar/src/modules/cb_groups.erl

```diff
@@ -154,7 +154,7 @@ delete(Context, _) ->
 -spec create(cb_context:context()) -> cb_context:context().
 create(Context) ->
     OnSuccess = fun(C) -> on_successful_validation('undefined', C) end,
-    cb_context:validate_request_data(<<"groups">>, Context, OnSuccess).
+    cb_context:validate_request_data(<<"callflows.ring_group">>, Context, OnSuccess).
```

Comment:
yes, it seems that it worked like this from the beginning, which means it doesn't validate anything.
i didn't look yet in `callflows.ring_group`  so confirm that it should be this one. `callflow.*`schemas are supposed to validate what is passed to cf_actions.